project-management/iteration-creation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Iteration creation
+description: These are the steps to follow when creating a new iteration
+---
+
+## Create a milestone
+
+Go to your squads repo and create a milestone titled: Iteration {number}-{week} {date from} - {date to} for example: Iteration 18-47 20th November - 3rd December.
+
+## Create a small task epic
+
+Create a new epic on the squad repo and attach it to the new milestone.
+
+## Set up the backlog
+
+Firstly, rename the sections of the backlog from `Current iteration:` to `Previous iteration`. Then rename `Next iteration` to `Current iteration`. Then create a new row titled `Next iteration` below `Current iteration.
```

Comment:
I think you should use quote here instead of backticks. Backticks are for code. Also, you're missing a closing backtick at the end.